src/plugins/intel_npu/src/compiler/src/zero_compiler_in_driver.cpp

```diff
@@ -453,13 +453,19 @@ std::string LevelZeroCompilerInDriver<TableExtension>::serializeConfig(
         content = std::regex_replace(content, std::regex(ov::intel_npu::tiles.name()), "NPU_DPU_GROUPS");
     }
 
-    // Batch mode property is not supported in versions < 5.5 - need to remove it
+    // Batch mode and cpu pinning property is not supported in versions < 5.5 - need to remove it
     if ((compilerVersion.major < 5) || (compilerVersion.major == 5 && compilerVersion.minor < 5)) {
```

Comment:
Is 5.5 the first version supporting this property?

Reply:
yes, verified. Actually there is a huge overlap, due to the fact that minor version wasn't bumped up when this prop was added. Prop was added last november, minor was bumped to 5.5 last month

Reply:
Talked more in private, turns out version 5.2.0 is the right threshold.